src/PhpSpreadsheet/Worksheet/Worksheet.php

```diff
@@ -1214,17 +1217,14 @@ public function getCell($pCoordinate, $createIfNotExists = true)
             }
         }
 
-        // Uppercase coordinate
-        $pCoordinate = strtoupper($pCoordinate);
-
         if (Coordinate::coordinateIsRange($pCoordinate)) {
```

Comment:
None of these checks actually need to use the uppercase version as they are only checking for symbols, not letters.